### PR TITLE
BugFix for PrivateLinkServiceConnectionState and onboard CosmosDB private endpoint connection type

### PR DESCRIPTION
**Pull request by bowen0106**
Fix a typo bug in PrivateLinkServiceConnectionState.
- mapping ActionsRequired to ActionRequired to avoid breaking changes.

Onboard CosmosDB type for PrivateEndpointConnection
- no breaking changes or cmdlets changes.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network/Common/NetworkResourceManagerProfile.cs [hunk 1]
```diff
@@ -1269,7 +1269,10 @@ private static void Initialize()
                 cfg.CreateMap<MNM.PrivateLinkServiceConnection, CNM.PSPrivateLinkServiceConnection>();
 
                 cfg.CreateMap<CNM.PSPrivateLinkServiceConnectionState, MNM.PrivateLinkServiceConnectionState>();
-                cfg.CreateMap<MNM.PrivateLinkServiceConnectionState, CNM.PSPrivateLinkServiceConnectionState>();
+                cfg.CreateMap<MNM.PrivateLinkServiceConnectionState, CNM.PSPrivateLinkServiceConnectionState>().AfterMap((src, dest) =>
```
Comment (number213) [inline src/Network/Network/Common/NetworkResourceManagerProfile.cs]: I'd say you should map in the other way too

**Comment by v-shhatt** [comment]
> Reviewed, please fix failure checks

Please fix below errors: Commands.Network.Test.ScenarioTests.PrivateLinkServiceTests.TestPrivateEndpointConnectionCRUDforCosmosDB [1ms]
2020-02-10T18:53:28.5529707Z EXEC : error Message:  [D:\a\1\s\build.proj]
2020-02-10T18:53:28.5530712Z      System.ArgumentException : Unable to find recorded mock file 
'D:\a\1\s\src\Network\Network.Test\bin\Debug\netcoreapp2.2\SessionRecords\Commands.Network.Test.ScenarioTests.PrivateLinkServiceTests\TestPrivateEndpointConnectionCRUDforCosmosDB.json'.

**Comment by bowen0106** [comment]
Hi @number213 , thanks for reviewing and sorry , i don't understand what is issue is . I just created this test file from latest commit status. Can you help to take a look ?
Also, I think there might be some bugs in the PS framework since I always got error when running Get-AzResource with resource group parameter in test file. I have to take some alternative way to resolve the issue.

**Comment by number213** [comment]
Hi, @bowen0116, if you want to make it into march release please resolve existing issues with CI

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**